demo/data-loading-demo.html

```diff
@@ -15,44 +15,20 @@
   <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
 
   <title>carbon-route data loading example</title>
-  <script src="../../../@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
-
-  <script type="module" src="../../iron-demo-helpers/url-bar.js"></script>
-  <script type="module" src="../../iron-demo-helpers/demo-pages-shared-styles.js"></script>
-  <script type="module" src="./data-loading-demo/flickr-search-demo.js"></script>
-
-  <!-- FIXME(polymer-modulizer):
-        These imperative modules that innerHTML your HTML are
-        a hacky way to be sure that any mixins in included style
-        modules are ready before any elements that reference them are
-        instantiated, otherwise the CSS @apply mixin polyfill won't be
-        able to expand the underlying CSS custom properties.
-        See: https://github.com/Polymer/polymer-modulizer/issues/154
-        -->
-    <script type="module">
-const $_documentContainer = document.createElement('template');
-$_documentContainer.setAttribute('style', 'display: none;');
-
-$_documentContainer.innerHTML = `<style is="custom-style" include="demo-pages-shared-styles">
-  </style>`;
-
-document.head.appendChild($_documentContainer.content);
-</script>
+  <script src="https://unpkg.com/@webcomponents/webcomponentsjs@^2.0.0/webcomponents-loader.js"></script>
+  <script type="module">
+    import '@polymer/iron-demo-helpers/url-bar.js';
+    import './data-loading-demo/flickr-search-demo.js';
+    import {html} from '@polymer/polymer/lib/utils/html-tag.js';
+    const template =
+        html`<style is="custom-style" include="demo-pages-shared-styles">
```

Comment:
doesn't this need to be wrapped in a `<custom-style>` for it to actually work in 2.0+?

Reply:
this style is never acutally used